### PR TITLE
Settings reference Review / Improvements

### PR DESCRIPTION
**Pull request by davegson**

### CODE REVIEW

**Comment by davegson** [comment]
good from my end - but please BS check my changes before merging

File: _data/config-manual.yml [hunk 1]
```diff
@@ -173,17 +176,17 @@ filter/preventBypassing:
...
 filter/removeOutOfScopeDNS:
   layered: true
   manual: |
-    [DNS Rebinding](https://en.wikipedia.org/wiki/DNS_rebinding) attacks allow an attacker to circumvent security policies. This feature blocks DNS Rebinding attacks on local systems.
+    [DNS Rebinding attacks](https://en.wikipedia.org/wiki/DNS_rebinding) allow an attacker to circumvent security policies. This feature blocks DNS Rebinding attacks on local systems.
 
 filter/serviceEndpoints:
   layered: true
   manual: |
-    If you need to accept incoming connections, try to narrow down from where you need to accept connections. A great way to start is to only accept connections from an organization via its AS number ([find it here](https://www.ultratools.com/tools/asnInfo)) or from one to a few countries.
```
Comment (dhaavi) [inline _data/config-manual.yml:186]: I'd find that link very helpful.

**Review by dhaavi — CHANGES_REQUESTED** [review]
Content:

- Everything looks good!

Design:

- Mobile support seems to be missing completely.
- I find the backticks for code section confusing, especially as they are only used sometimes.
- Red URLs are really weird.
- Green ones less so, but also.
- Having no differing design elements or icons makes it hard to distinguish between the different labels assigned to settings.
- Each setting could use more spacing to the next setting. Everything seems like one big wurst.
- Intendation in the menu is too little in order to quickly see the levels.
- Font could be thinner and darker in order to make it easier to read - like the one in the code blocks.
- I'd really want to revert d2b79fbc0ef16f8052026e30ff0a3a0c4296a9d2, as this is _very_ helpful for support.

All in all, this turned out very nicely and I can't to see it live!

Also, there are pending updates to the settings, which I will add when https://github.com/safing/portmaster/pull/281 is reviewed and merged.

File: _data/config-manual.yml (hunk 1 above)
Comment (davegson) [reply _data/config-manual.yml:186]: that link was broken on my end

Comment (davegson) [reply _data/config-manual.yml:186]: https://github.com/safing/docs/pull/12/commits/69c270480090301f4c84a057a0c960399e1ed20c

but please add a working one if you can

**Comment by davegson** [comment]
many of the design aspects are being tackled in https://github.com/safing/docs/pull/8 - I noted those which are missing and will address them with Luke.

For now it'd be nice to merge the PR.

File: _data/config-manual.yml (hunk 1 above)
Comment (dhaavi) [reply _data/config-manual.yml:186]: I found this one: https://bgpview.io/

**Pull request merged**